cfgov/regulations3k/jinja2/regulations3k/regulations3k-secondary-navigation.html

```diff
@@ -43,18 +43,12 @@
             Table of Contents
         </span>
         <span class="o-expandable_header-right o-expandable_link">
-            <span class="o-expandable_cue o-expandable_cue-open">
-                <span class="u-visually-hidden">
-                    {{ _('Show') }}
-                    {{ expandable_cue_additional_text }}
-                </span>
+            <span class="o-expandable_cue o-expandable_cue-open"
+                  aria-label="{{ _('Show') }} {{ expandable_cue_additional_text }}">
```

Comment:
I'm not sure if these will be read, since `span` is not an interactive element. https://developer.paciellogroup.com/blog/2017/07/short-note-on-aria-label-aria-labelledby-and-aria-describedby/

Given the reported inconsistency between browser/screenreader combos, might be safer to leave these as-is.

Reply:
Okay I'll revert it. Perhaps the svg_icon shortcode should provide a means to pass in a `title`, per the instructions in inline SVG on https://css-tricks.com/accessible-svgs/

Reply:
Yeah, I think that'd be a good idea.